content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -245,49 +279,44 @@ imagePullSecrets:
 - name: myregistrykey
 ```
 
-Finally replace the serviceaccount with the new updated `sa.yaml` file
-
-```shell
-kubectl replace serviceaccount default -f ./sa.yaml
-```
-
-Now, any new pods created in the current namespace will have this added to their spec:
+Now you've made those changes, any new Pods created in the current namespace
+will have this added to their spec:
```

Comment:
nit: Do you need: Now that you've made those changes? Above is, `After you've made those changes`. Could just say, `Any new Pods ...`